src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/index.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import {css} from '@emotion/core';
+import styled from '@emotion/styled';
+
+import {ModalRenderProps} from 'app/actionCreators/modal';
+import {Client} from 'app/api';
+import Button from 'app/components/button';
+import TextOverflow from 'app/components/textOverflow';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization, Project} from 'app/types';
+import {Image} from 'app/types/debugImage';
+import theme from 'app/utils/theme';
+import withApi from 'app/utils/withApi';
+
+import NotAvailable from './notAvailable';
+import Panel from './panel';
+
+const UPLOADED_TO_SENTRY = 'sentry:project';
+
+type Props = ModalRenderProps & {
+  api: Client;
+  projectId: Project['id'];
+  organization: Organization;
+  image: Image;
+  title?: string;
+};
+
+function DebugFileDetails({
+  Header,
+  Body,
+  Footer,
+  closeModal,
+  title,
+  image,
+  organization,
+  api,
+  projectId,
+}: Props) {
+  const {debug_id, image_addr, arch: architecture, code_file, candidates = []} = image;
+
+  const uploadedCandidates = candidates.filter(
+    candidate => candidate.source === UPLOADED_TO_SENTRY
+  );
+
+  const externalCandidates = candidates.filter(
+    candidate => candidate.source !== UPLOADED_TO_SENTRY
+  );
+
+  // TODO(PRISCILA): Should we perform a new request to fetch the debug files?
+  // because at the moment the payload of the event is not being updated
+  // async function handleDelete(debugId: string) {
+  //   api.request(
+  //     `/projects/${organization.slug}/${projectId}/files/dsyms/?id=${debugId}`,
+  //     {
+  //       method: 'DELETE',
+  //       complete: () => {
+  //
+  //       },
+  //     }
+  //   );
+  // }
+
+  return (
+    <React.Fragment>
+      <Header closeButton>{title ?? t('Unknown name')}</Header>
+      <Body>
+        <Content>
+          <GeneralInfo>
+            <Label>{t('Address Range')}</Label>
+            <Value>{image_addr}</Value>
```

Comment:
I think some images do not have address ranges (wasm), does this account for that?